scripts/action/update.sh

```diff
@@ -3,12 +3,16 @@
 version=""
 
 while IFS= read -r -d '' f; do
-  v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*")' "${f}" | sed -e 's/.*@//g')"
+  v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*") | . + " # " + line_comment' "${f}" | sed -e 's/.*@//g')"
   if [ -n "${v}" ]; then
+    if [[ "${v}" == *" # v"* ]]; then
+        v="${v#* # }"
+    fi
```

Comment:
```suggestion
  v="$(yq '.jobs.*.steps[].uses | select(. == "super-linter/super-linter*") | line_comment' ${f})"
  if [ -n "${v}" ]; then
```

yqの時点でコマンドから取得してくる感じにしてはダメかしら？

Reply:
これって他のところで呼び出して使う場合には、そこにあるやつが使われる？
つまり、hashで書かれたやつと、バージョンで書かれたやつがある可能性があり、両方に対応しないとなと思ってた。